docs/_documentations/troubleshooting.md

```diff
@@ -158,6 +158,43 @@ You might occasionally see projects stuck in the `Starting` or `Stopped` state e
 1. In the **Codewind Explorer** view, right-click your project and select **Build**.
 2. Wait for the project state to return to **Running** or **Debugging** in the **Codewind Explorer** view.
 
+<!--
+Action/Topic: Creating a project and/or Checking the application and build statuses
+Issue type: bug/info
+Codewind version: 0.9.0
+Issue link: https://github.com/eclipse/codewind/issues/1269
+-->
+## How to stop the app from continuously pinging 
+With all the stacks and templates Codewind offers, some template applications come with no server, like Node.js. The backend in PFE expects all applications to have a server. Thus, the backend continuously pings the port of the application retrieved from the application's container information. Since no server is available to ping on that port, the application times out and is stuck on `Starting`. 
```

Comment:
Can we make it clear that it is the `Appsody Node.js` that does not have a server? All codewind default templates have servers and it is just that some appsody ones do not have server